### PR TITLE
[14.0][MIG] partner_employee_quantity: Migration to 14.0

### PR DESCRIPTION
**Pull request by newtratip**
Standard Migration, Ready for review.

### CODE REVIEW

**Review by rousseldenis — APPROVED** [review]

**Comment by newtratip** [comment]
@ecosoft-odoo Could you review this ?

**Review by rousseldenis — APPROVED** [review]
Code review

File: partner_employee_quantity/views/res_partner_view.xml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    <record id="view_partner_form" model="ir.ui.view">
+        <field name="name">Partner form with employees</field>
+        <field name="model">res.partner</field>
+        <field name="inherit_id" ref="base.view_partner_form" />
+        <field name="arch" type="xml">
+            <page name="internal_notes" position="inside">
+                <group>
+                    <group
+                        name="employee_quantity"
+                        string="Employee quantity"
```
Comment (NL66278) [inline partner_employee_quantity/views/res_partner_view.xml:12]: Why specify string here? Would be the same as on the field?

**Comment by NL66278** [comment]
It should not be to much work to add the minimum and maximum quantity to the range, and make the range on partner a computed field based on the number of employees. Given that the module is designated as "Mature", this would be kind of expected?

**Review by HaraldPanten — APPROVED** [review]

**Pull request merged**